src/main/java/io/github/dftrakesh/cloverrest/model/updateInventory/updateRequest.java

```diff
@@ -0,0 +1,11 @@
+package io.github.dftrakesh.cloverrest.model.updateInventory;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Data;
+
+@Data
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class updateRequest {
```

Comment:
change class name updateRequest to InventoryRequest

Reply:
Updated name